hazelcast/src/main/java/com/hazelcast/sql/SqlColumnType.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.sql;
+
+import javax.annotation.Nonnull;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.OffsetDateTime;
+
+/**
+ * SQL column type.
+ */
+public enum SqlColumnType {
+    /** VARCHAR type, represented by {@link java.lang.String} */
```

Comment:
does this qualify as standard Javadoc format so it's picked up by the generator?

Reply:
Of course...

Reply:
From the Oracle docs [1]:
> This tag is valid in all documentation comments: overview, package, class, interface, constructor, method and **field**

[1] https://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html#CHDDIECH

Reply:
Resolving for now. Please re-open the conversation if you foresee any problems with these JavaDocs.